graph/encoding/dot/decode.go

```diff
@@ -26,6 +26,17 @@ type DOTIDSetter interface {
 	SetDOTID(id string)
 }
 
+// PortSetter is implemented by graph.Edge and graph.Line that can set
+// the DOT port and compass directions of an edge.
+type PortSetter interface {
+	// SetFromPort sets the From port and
+	// compass direction of the receiver.
+	SetFromPort(port, compass string) error
+	// SetToPort sets the To port and compass
```

Comment:
Final nit. Please place a blank line between the first method and the following comment.